tests/console/systemd_testsuite.pm

```diff
@@ -18,12 +18,14 @@ use utils;
 use version_utils qw(is_leap is_tumbleweed is_sle);
 
 sub run {
-    my $qa_head_repo = get_required_var('QA_HEAD_REPO');
-    if (is_leap('=15.0')) {
-        $qa_head_repo = 'https://download.opensuse.org/repositories/devel:/openSUSE:/QA:/Leap:/15/openSUSE_Leap_15.0/';
-    }
-    elsif (is_sle('=15')) {
-        $qa_head_repo = 'http://download.suse.de/ibs/QA:/SLE15/standard/';
+    my $qa_head_repo = get_var('QA_HEAD_REPO', '');
+    if (!$qa_head_repo) {
+        if (is_leap('15.0+')) {
```

Comment:
is there any difference between '15.0+' and '15+' ? For me both saying from SLE 15 SP0 and all above.

Reply:
There is a different implementation on **is_leap** an **is_sle**. I tried to use the same symbol for both without success.